.github/workflows/release-iii.yml

```
@@ -487,6 +487,25 @@ jobs:
       SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
       SLACK_CHANNEL_ID: ${{ secrets.SLACK_CHANNEL_ID }}
 
+  sdk-go:
+    name: SDK Go Publish
+    needs: [setup, create-iii-release]
+    if: ${{ !failure() && !cancelled() }}
+    # Go has no registry: this pushes the subdirectory-scoped module tag
+    # (sdk/packages/go/iii/vX.Y.Z) in lockstep with the engine release, so
+    # `go get` can resolve the module. No registry token required.
+    uses: ./.github/workflows/_go.yml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/_go.yml": open /home/jailuser/git/.github/workflows/_go.yml: no such file or directory

(workflow-call)